src/test/java/org/carlspring/cloud/storage/s3fs/util/CopyDirVisitor.java

```diff
@@ -69,12 +69,12 @@ private Path appendPath(Path to)
     {
         Path targetPath = toPath;
 
-        // sacamos el path relativo y lo recorremos para
-        // añadirlo al nuevo
+        // Take out the relative path and traverse through it
+        // and add new to it..
         for (Path path : fromPath.relativize(to))
         {
-            // si utilizamos path en vez de string: lanza error por ser
-            // distintos paths
+            // If we use path instead of string : It Throws an error
+            // due to the different paths..
```

Comment:
I would use: "If Path is used instead of String, an error is thrown because the paths are different."

Reply:
:+1: Thanks, @ptirador!

@viveksahu26: Could you please apply this change?